Resources/views/Collector/db.html.twig

```diff
@@ -25,7 +43,7 @@
         </div>
         <div class="sf-toolbar-info-piece">
             <b>Invalid entities</b>
-            <span class="sf-toolbar-status sf-toolbar-status-{{ collector.invalidEntityCount > 0 ? 'red' : 'green' }}">{{ collector.invalidEntityCount }}</span>
+            <span class="sf-toolbar-status sf-toolbar-status-{{ collector.invalidEntityCount > 0 ? 'red' }}">{{ collector.invalidEntityCount }}</span>
```

Comment:
I think I finally did it here: https://github.com/doctrine/DoctrineBundle/commit/461e2e35669d793cb91fe9f39ff28c33800e5fb1